client/app/styles/tip.css

```diff
@@ -1,6 +1,7 @@
 .TipPageSectionTitle {
-  font-size: 20px;
-  font-weight: bold;
+  font-size: 17px;
+  font-weight: bolder;
+  font-variant: small-caps;
```

Comment:
nice change the 'small-caps' but lets first evaluate if it's ok to use it for every title all along the application and keep things uniform to not create confusion.

i'm also thinking to find some good product and copying its style as a whole :)